app/models/photo.rb

```diff
@@ -0,0 +1,15 @@
+class Photo < ActiveRecord::Base
+
+    has_attached_file :image, styles: { medium: "700x700>", thumb: "100x100>" }, default_url:  "/images/:style/missing.png"
+    validates_attachment_content_type :image, :content_type => /\Aimage\/.*\Z/
+
+    belongs_to :user
+    validates_presence_of :image
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.